GetIntoTeachingApi/Models/GetIntoTeaching/Validators/TeachingEventUpsertOperationValidator.cs

```diff
@@ -12,10 +12,14 @@ public TeachingEventUpsertOperationValidator(ICrmService crm)
         {
             _crm = crm;
 
+            // The regex we use here has been relaxed be due to a number of
+            // pre-existing events that would otherwise fail the validation.
+            // We should look to revert it back to the stricter regex in the
+            // future (when these events have since passed - 24/6/21).
```

Comment:
I think this should be 2022. Wondering whether it's worth putting a `TODO: ` in so that this is picked up by SonarCloud and not left to be forgotten?

Reply:
Whoops. Yep.